test/e2e/serial/tests/configuration.go

```diff
@@ -1071,12 +1071,14 @@ var _ = Describe("[serial][disruptive] numaresources configuration management",
 			pods, err := podlist.With(fxt.Client).ByDeployment(ctx, updatedDeployment)
 			Expect(err).ToNot(HaveOccurred())
 
-			// Here we have an assertion for the length of the pods to be smaller than the maxPodsWithTAE.
 			// Based on a couple of test runs it accurs that some pods sometimes may be created in cases that the time frame
 			// between the kubeletchanges and the deployment being created are too small for the scheduler to pick up the new changes and recover
-			// (sometimes even a second is too much) and it can create pods with Topology Affinity Error that have zero effect on the deployment itself
-			// and theirs status is ContainerStatusUnknown. the count should always be bounded and no more than the maxPodsWithTAE const.
-			Expect(len(pods)).To(BeNumerically("<", maxPodsWithTAE))
+			// (sometimes even a second is too much), and it can create pods with Topology Affinity Error that have zero effect on the deployment itself
+			// and theirs status is ContainerStatusUnknown, the count of these pods needs to be bounded once the pending pod comes up.
+			// The conclusion is as long as there is exactly one pod that is pending the scheduler is behaving correctly.
+			if len(pods) > maxPodsWithTAE {
+				klog.Warningf("pods list is bigger than %d", maxPodsWithTAE)
```

Comment:
Let's add the actual number  deployments pods that we got in the same warning

Reply:
np will add